docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,83 @@
+Running the Application
+-----------------------
+
+Before we run the application we have to first build it. Let's take a look at the build tool in action for this project: [sbt](http://www.scala-sbt.org). _If you prefer to run the application from an IDE instead of a Terminal window you should look at the @ref:[IntelliJ IDEA](intellij-idea.md) section._
+
+### The Build Files
+
+sbt uses a `build.sbt` file to handle the project. This project's `build.sbt` file looks like this:
+
+@@snip [build.sbt]($g8root$/build.sbt)
+
+This build file is very simple. In essence, it creates one project `hello-akka-scala` and declares the dependencies for this project. We must also declare what version of sbt to use and this is done in the file `project/build.properties`:
+
+@@snip [build.properties]($g8root$/project/build.properties)
+
+Now we are ready to build and run the project.
+
+### Compiling the Project
+
+We will use sbt to handle the compilation and run. We open a Terminal window and, from inside the project directory, type the following:
+
+```
+$ sbt
```

Comment:
I think that information like this should go here instead: http://developer.lightbend.com/start/

This way we have the information in one place and we don't have to add the information to the projects themselves. What do you think about this idea @eed3si9n?